todo_app/app.py

```diff
@@ -115,55 +20,194 @@ def __init__(self, list, name, desc):
 
     def add_card(self):
         #add a new card to a list
+        key = "key=" + os.getenv('API_KEY')
+        token = "token=" + os.getenv('API_TOKEN')
+        return ("https://api.trello.com/1/cards?idList=" + self.list + "&name=" + self.name + "&desc=" + self.desc + "&" + key + "&" + token)
+
+class ViewModel:
+    def __init__(self, items):
+        self._items = items
+    @property
+    def items(self):
+        return self._items
+
+    @property
+    def todo_items(self):
+        return_list = []
+        #self.items = list of dicts
+        for x in self._items:
+            #x = dict
+            for item in x.items():
+                #item = each item in the dict
+                if item[0] == "status" and item[1] == 'To Do':
+                    return_list.append(x)
```

Comment:
This works nicely, but could be simplified; since we know which field in the dictionary we're looking for we don't need to iterate over all of them:
```suggestion
        for x in self._items:
            #x = dict
            status = x.get('status') # could be x['status'] if you're confident the card will have a status field
            if status == 'To Do':
                    return_list.append(x)
```